packages/components/src/RichTooltip/RichLayout/RichLayout.component.js

```diff
@@ -60,4 +64,5 @@ RichLayout.propTypes = {
 	Header: Inject.getReactElement.propTypes,
 	Footer: Inject.getReactElement.propTypes,
 	text: PropTypes.string,
+	prefixId: PropTypes.string.isRequired,
```

Comment:
WDYT? to be consistent!
```suggestion
	id: PropTypes.string.isRequired,
```

Reply:
I expect that if i name the variable "id", i do id={id} in the component. In my case, I use a prefix to create the id used by the component, so, why i name it prefixId. However, i don't disturb me to change to id if you prefer.

Reply:
If it bothers you, you can implement the behavior you just described, adding the id on the main element wrapper, and still use this id to generate ids on sub parts

Reply:
+1 yes it sounds great